cg/services/order_validation_service/models/existing_sample.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel, Field
+
+from cg.models.orders.sample_base import NAME_PATTERN, StatusEnum
+
+
+class ExistingSample(BaseModel):
+    father: str | None = Field(None, pattern=NAME_PATTERN)
+    internal_id: str
+    mother: str | None = Field(None, pattern=NAME_PATTERN)
+    status: StatusEnum | None = None
```

Comment:
Had to make this optional in the end since we are using this for existing microsalt samples as well.

Reply:
I don't think existing microsalt samples exist...

Reply:
Removed!